sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/DigitalTwinsAsyncClient.java

```diff
@@ -816,24 +828,24 @@ public DigitalTwinsServiceVersion getServiceVersion() {
     }
 
     /**
-     * Gets the list of models by iterating through a collection.
-     * @param listModelOptions The options to follow when listing the models. For example, the page size hint can be specified.
-     * @return A {@link PagedFlux} of ModelData and the http response.
+     * List all of the models in this digital twins instance.
+     * @return A {@link PagedFlux} of {@link ModelData} that enumerates all the models.
      */
     @ServiceMethod(returns = ReturnType.COLLECTION)
-    public PagedFlux<ModelData> listModels(ListModelOptions listModelOptions) {
-        return new PagedFlux<>(
-            () -> withContext(context -> listModelsSinglePageAsync(listModelOptions, context)),
-            nextLink -> withContext(context -> listModelsNextSinglePageAsync(nextLink, context)));
+    public PagedFlux<ModelData> listModels() {
```

Comment:
the changes from lines 819 to 827 are just swapping the order of these two list APIs. Now the minimal list API is above the maximal list API for consistency